src/constructs/core/parameters/acm.ts

```diff
@@ -1,15 +1,14 @@
 import { RegexPattern } from "../../../constants";
+import { AppIdentity } from "../identity";
 import type { GuStack } from "../stack";
-import type { GuNoTypeParameterProps } from "./base";
 import { GuStringParameter } from "./base";
 
 export class GuCertificateArnParameter extends GuStringParameter {
-  constructor(scope: GuStack, id: string = "TLSCertificate", props?: GuNoTypeParameterProps) {
-    super(scope, id, {
+  constructor(scope: GuStack, props: AppIdentity) {
```

Comment:
This gives the ability to have a `GuCertificateArnParameter` for each `app` defined in a  stack 🎉 .